modules/ismp/clients/parachain/client/src/consensus.rs

```diff
@@ -67,7 +67,8 @@ pub struct ParachainConsensusProof {
 pub const PARACHAIN_CONSENSUS_ID: ConsensusClientId = *b"PARA";
 
 /// Slot duration in milliseconds
-const SLOT_DURATION: u64 = 12_000;
+const ASYNC_SLOT_DURATION: u64 = 6_000;
+const SYNC_SLOT_DURATION: u64 = 12_000;
```

Comment:
We shouldn't hardcode slot durations at all; they should be configured by the call that adds the parachain to the pallet and stored onchain, also, allow them to be updated because parachains can now choose their slot duration post async backing

https://github.com/polytope-labs/hyperbridge/blob/3f20eb2227557760b31b2baee7515b839c9ffc0e/modules/ismp/clients/parachain/client/src/lib.rs#L74-L75